src/phoenix/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class PhoenixException(Exception):
+    pass
```

Comment:
This error is only used for retries and token limits and feels overly generic. I think it's preferable to make it less generic to avoid cross-sub-module dependencies as much as possible.